static/js/directives/ui/header-bar.js

```diff
@@ -145,6 +145,14 @@ angular.module('quay').directive('headerBar', function () {
           'namespace': namespace
         };
       };
+
+      $scope.checkElementByID = function(elementID) {
+        var results = false;
```

Comment:
`results` suggest a multiple value like an array. `result` or even `exists` should work

Reply:
Fixed